.github/workflows/update-catch.yaml

```
@@ -0,0 +1,48 @@
+name: Update Catch
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 10 1/16 * *'
+jobs:
+  updateCatch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get Latest Catch Tagged Release
+        id: catch-header
+        run: |
+          # Ping version information upstream
```

```suggestion
          # Ping version information upstream.
```

Just for consistency.